test/AutomatedCarTests/SystemComponents/InputHandlerTests.cs

```diff
@@ -0,0 +1,383 @@
+﻿using AutomatedCar;
+using AutomatedCar.SystemComponents;
+using Avalonia.Input;
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+namespace Tests.SystemComponents
+{
+    public class InputHandlerTests : IDisposable
+    {
+        HumanMachineInterface hmi;
+
+        public InputHandlerTests()
+        {
+            hmi = new HumanMachineInterface(new VirtualFunctionBus());
+        }
+
+        public void KeyboardFill()
+        {
+            for (int i = 0; i < 173; i++)
+            {
+                Keyboard.Keys.Add((Key)i);
+            }
+            for (int i = 10001; i < 10005; i++)
+            {
+                Keyboard.Keys.Add((Key)i);
+            }
+        }
+
+        [Fact]
+        public void WhitoutKeyPressedAllVarialbeRemainsFalse()
+        {           
+            hmi.InputHandler();
+            Assert.False(hmi.ACC);
+            Assert.False(hmi.ACCDistance);
+            Assert.False(hmi.ACCSpeedMinus);
+            Assert.False(hmi.ACCSpeedPlus);
+            Assert.False(hmi.BreakpedalDown);
+            Assert.False(hmi.CameraDebug);
+            Assert.False(hmi.GaspedalDown);
+            Assert.False(hmi.GeerDown);
+            Assert.False(hmi.GeerUp);
+            Assert.False(hmi.LaneKeeping);
+            Assert.False(hmi.ParkingPilot);
+            Assert.False(hmi.RadarDebug);
+            Assert.False(hmi.SteeringLeft);
+            Assert.False(hmi.SteeringRight);
+            Assert.False(hmi.TurnSignalLeft);
+            Assert.False(hmi.TurnSignalRight);
+            Assert.False(hmi.UtrasoundDebug);
+        }
+
+        [Fact]
+        public void WhitBadInpustGaspedalStaysFalse()
```

Comment:
_WhitBadInpustValami_ helyett _**With**Bad**Inputs**Valami_, es az alatta levoknel is ugyanigy el van irva a legtobb helyen (ez sem logika hiba, csak eliras)